django_auth_lti/middleware_patched.py

```diff
@@ -1,19 +1,17 @@
 import logging
 
-import django_auth_lti.patch_reverse
-
 from django.contrib import auth
 from django.core.exceptions import ImproperlyConfigured
 from django.conf import settings
+from django.utils.deprecation import MiddlewareMixin
 
+from .conf import get_excluded_paths
 from .timer import Timer
-
 from .thread_local import set_current_request
-
-try:
-    from django.utils.deprecation import MiddlewareMixin
-except ImportError:  # Django < 1.10
-    MiddlewareMixin = object
+# importing here will ensure that django.urls.reverse is patched
+# for other libraries and parts of django, e.g. `url` template tag,
+# when the middleware is loaded
+import django_auth_lti.patch_reverse
```

Comment:
Thanks for highlighting this. I wouldn't have given a second thought to the change in import order had this not been there.